_data/glossary.yaml

```diff
@@ -219,6 +221,8 @@ parent image: |
   in the image's Dockerfile. All subsequent commands are applied to this parent
   image. A Dockerfile with no `FROM` directive has no parent image, and is called
   a **base image**.
+persistent storage: |
+  Persistent storage or volume storage provides a way for a user to add a persistent layer to the running container's file system. This persistent layer could live on the container host or an external device. The lifecycle of this persistent layer is not connected to the life cycle of the container, allowing a user to retain state.
```

Comment:
This definition uses both "lifecycle" and "life cycle". They're both technically right, but can we decide on one?

Reply:
Yes, it should be "lifecycle." @DawnWood-Docker Can we update this? Thanks.

Reply:
@traci-morrison yes